admin/.storybook/main.js

```diff
@@ -19,11 +19,14 @@ const reactIntlTransformRule = {
   exclude: /node_modules/,
 };
 
+const isMerged = (process.env.MERGE_STORYBOOKS === 'true');
```

Comment:
Does the `MERGE_STORYBOOKS` need to get added to the admin/.env file? If that's the case we should add it to the .env/example file.

Reply:
Nvm, just found out that it defaults to false because its not in our .env files, and its only the github action workflow that sets this variable.

Reply:
I actually think it's a great idea to keep it in a `admin/.env.example`! I like the idea of one file that's am authoritative map for all the envvars the code uses :)